### ISSUE TITLE
Too much redundant information in City tool tips hides important info

### ISSUE THREAD

**Issue by UndeadDevel** [LoneGazebo/Community-Patch-DLL#5073]
_**1. Mod version** (i.e Date - 4/23):_
11-30-18

_**2. Mod list** (if using Vox Populi only, leave blank):_

VP (EUI)
City-States Leader for VP (v8)
List Enhancements for VP (v4.2)
Trade Opportunities for VP (v8.1)
Reforestation (v8) (slightly modified by me)
Civ-linked GG/GA (v11)
Enhanced Naval Warfare for VP (v1.6)
Enlightenment Era (VP) (v1.2)
Faster Aircraft Animations (v3)
Capture Great People (I neutered this one because of compatibility issues)
Espionage Mapping (v7) (modified by me)
Religious Settlers (v1)
Improved Air Recon (v1)
Missiles No Supply (v1)
More Unique Components for VP (v38)
Promotions Flags EUI (v1.1) (slightly modified by me)
Subs Ignore Borders (v3)
Subs Immune Under Ice (v2)
Destination (v4)
Filters in Trade Route Overview (VP) (v1.3) (slightly modified by me)
Improved City View (VP EUI) (v11.0)
Promotion Tree (v17)
Upgrade Tree (v9)
VP Promo Icons (v1.9)

_**3. Error description**:_
When mousing over a foreign city of a civ that the player is at war with, the tool tip can grow so large that important information is not shown because the tool tip length exceeds screen height; see screenshot below.
Most of the information in the tool tip is already shown in the leader list to the right (with EUI), so it is redundant in the city tool tip; the liberation bonuses, on the other hand, cannot be viewed anywhere else.

If the redundant info cannot be removed (maybe it is needed there for people who don't use EUI, I dunno since I don't remember if that info is shown anywhere else without EUI) then the conquest and liberation info should be moved up top in the tool tip, since it is unique for each city while the other info is the same for all cities of that leader and therefore can be viewed in other cities that don't have the option of liberation and thus a smaller tool tip.

Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. Make sure you have enabled logging before experiencing an error! Go here to find out how: http://forums.civfanatics.com/showthread.php?t=487482):

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves):

3. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steamapps/common/Sid Meier's Civilization V):

4. Screenshots (optional):
![city_tool_tips](https://user-images.githubusercontent.com/30495439/51343682-d0118c80-1a8e-11e9-82ce-0711910586ca.jpg)

**Comment by ryanmusante** [comment]
Isn't that transparent diplomacy? I have that disabled by default.

**Comment by Vyyt02** [comment]
Sorry for derailing, but what is that naval unit right under Hilo (belongs to Morocco) and where do the nice graphics for the destroyer come from? It is not from ENW, is it?

**Comment by UndeadDevel** [comment]
@ryanmusante Nope, TD is turned off. There are no mods or anything else (beside VP and maybe EUI) influencing the tool tip in this game.

@Vyyt02 Better to ask the question in the forum and link here...but I'll answer quickly: The Moroccan unit is the Corsair from MUCfVP; the Destroyer is from ENW, yes; I have made the Cruiser and Ironclad smaller, though, so the destroyer doesn't look tiny in comparison.

**Comment by LoneGazebo** [comment]
Agreed. Fixed.

**Comment by Vyyt02** [comment]
@UndeadDevel Thanks, I know, but I don't know your nickname on CivFanatics, so I don't know how to ask you there. Sorry, I was unclear, I meant the badge/icon of the destroyer in the left bottom, not the 3D model. </EndOfDerailing>

**Comment by UndeadDevel** [comment]
No idea what you mean; if you're talking about the large, round 2D image of the Destroyer directly above the Explore and Delete buttons, then I assume that's the standard ENW image for that unit (if it isn't I don't know where it comes from, either, but you can see all mods used in the OP). If you mean the promotion icons then those are from VP Promo Icons.
I'm also using EEVP, so maybe something is interacting there...no clue.

**Comment by Vyyt02** [comment]
@UndeadDevel, thanks, yes, I am talking about the big 2D image. Maybe my ENW is outdated or, as you say, it is something from EEVP.
